python/lsst/ip/diffim/utils.py

```diff
@@ -1173,6 +1173,70 @@ def evaluateMeanPsfFwhm(exposure: afwImage.Exposure,
     return np.nanmean(width)
 
 
+def computeAveragePsf(exposure: afwImage.Exposure,
+                      psfExposureBuffer: float, psfExposureGrid: int) -> float:
+    """Get the average PSF by evaluating it on a grid within an exposure.
+
+    Parameters
+    ----------
+    exposure : `~lsst.afw.image.Exposure`
+        The exposure for which the average PSF is to be computed.
+        The exposure must contain a `psf` attribute.
+    psfExposureBuffer : `float`
+        Fractional buffer margin to be left out of all sides of the image
+        during the construction of the grid to compute average PSF in an
+        exposure.
+    psfExposureGrid : `int`
+        Grid size to compute the average PSF in an exposure.
+
+    Returns
+    -------
+    psfImage : `np.ndarray`
+        The average PSF across the exposure.
+
+    Raises
+    ------
+    ValueError
+        Raised if the PSF cannot be computed at any of the grid points.
+
+    See Also
+    --------
+    evaluateMeanPsfFwhm
```

Comment:
Could you also backlink this from `evaluateMeanPsfFwhm`? And while we are modifying the docstring of `evaluateMeanPsfFwhm`, could you change the first line there to say mean instead of median?